roles/blockscout/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+user: ubuntu
+
+allow_blockscout_ssh: true
+allow_blockscout_p2p: true
+allow_blockscout_http: false
+
+username: "blockscout"
+users:
+- name: "{{ username }}"
+home: "/home/{{ username }}"
+
+MAIN_REPO_FETCH: "poanetwork"
+MOC_ADDRESS: "0xdd0bb0e2a1594240fed0c2f2c17c1e9ab4f87126"
+
+reboot_after_completion: false
+install_debug_tools: "no"
+install_firewall: false
+
+blockscout_version: "df6ec81b68"
+db_user: "blockscout"
+db_user_password: "dvYYTbhdhuuOLMf74ghu"
```

Comment:
Let's not have defaults for password, it's better to fail

Reply:
I've removed password from defaults and made empty in `group_vars/blockscout.yml.example`, as this file is copied then to settings. It will cause error now any way, either for undefined variable or authentication error on db creation (with empty string).